dicodile/data/tests/test_text.py

```diff
@@ -0,0 +1,8 @@
+from pathlib import Path
+
+
+def test_data_dir(monkeypatch):
+    monkeypatch.setenv("DICODILE_DATA_HOME", "/home/unittest/dicodile")
+
+    from dicodile.data.text import DATA_DIR
+    assert DATA_DIR == Path("/home/unittest/dicodile/dicodile/images/text")
```

Comment:
Not sure this can work because it is defined when the module is imported first no? so if the `config` has already been imported, you won't change its value when modifying the env variable.

Reply:
Yes, I am fixing it now.

Reply:
Are we expecting users to import dicodile.data.text.DATA_DIR? If not, we might restrict tests to the API that we expect them to use? (e.g. get_XXX_data functions?)

Reply:
I added this test because codecov was complaining about uncovered lines. I do not know if there is another way to suppress that.

Reply:
@tomMoral Actually it is not a relevant test. Shall we ignore codecov complaints in such cases?

Reply:
Actually, I think `DATA_DIR` should be renamed `TEXT_DATA_DIR` as it is specific to text data.
I would say the test should only be done on `DATA_HOME`, checking that you can indeed modify it as you did.

Reply:
Then I'll remove this test and add a test in https://github.com/hndgzkn/dicodile/blob/dicodile_config/dicodile/tests/test_config.py to test `DATA_HOME` besides testing `get_data_home`?